sdk/python/orm/sqlalchemy.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: SQLAlchemy
+sidebarTitle: SQLAlchemy
+description: Configure SQLAlchemy to work with your Turso database
+---
+
+![SQLAlchemy Quickstart](/images/guides/sqlalchemy-banner.png)
+
+## Prerequisites
+
+Before you start, make sure you:
+
+- [Install the Turso CLI](/cli/installation)
+- [Sign up or login to Turso](/cli/authentication#signup)
+
+<Steps>
+
+<Step title="Install the libSQL dialect for SQLAlchemy">
+
+```bash
+pip install sqlalchemy-libsql
+```
+
+</Step>
+
+<Step title="Configure database credentials">
+
+Get the database URL:
+
+```bash
+turso db show --url <database-name>
+```
+
+Get the database authentication token:
+
+```bash
+turso db tokens create <database-name>
+```
+
+Assign credentials to the environment variables inside `.env`.
+
+```bash
+TURSO_DB_URL="..."
```

Comment:
We should try to be more consistent with the names of these environment variables. So far we've been using `TURSO_DB_URL` and `TURSO_DB_AUTH_TOKEN` in docs, but they're different for _some_ examples.

What if we changed these (and eventually all examples) to be more explicit:

```bash
TURSO_DATABASE_URL=
TURSO_AUTH_TOKEN=
```

The reason I've changed it to `TURSO_AUTH_TOKEN` is because the token could be a group OR database token.